### PR TITLE
russian translations for the reset password bundle

### PR DESCRIPTION
**Pull request by jrushlow**
_Translations were AI Generated by AWS Translate Machine Learning Platform and need to be Validated by humans_

Includes all of the Russian translations for Symfony Cast's Reset Password Bundle in PR #1100

### CODE REVIEW

File: translations/messages+intl-icu.ru.xlf [hunk 1]
```diff
@@ -438,6 +438,63 @@
...
+                <target>Забыли пароль</target>
+            </trans-unit>
+            <trans-unit id="reset_password.title">
+                <source>reset_password.title</source>
+                <target>Сброс пароля</target>
+            </trans-unit>
+
+            <trans-unit id="reset_password.check_email.p1">
+                <source>reset_password.check_email.p1</source>
+                <target>Отправлено электронное письмо, содержащее ссылку, которую можно щелкнуть, чтобы сбросить пароль. Срок действия этой ссылки истекает через {token} час (ы).</target>
```
Comment (andrewmy) [inline translations/messages+intl-icu.ru.xlf:453]:
```suggestion
                <target>Отправлено электронное письмо, содержащее ссылку, по которой можно сбросить пароль. Срок действия этой ссылки истекает через {token} {token, plural, =0 {часов} one {час} few {часа} many {часов} other {часов}}.</target>
```

File: translations/messages+intl-icu.ru.xlf [hunk 2]
```diff
@@ -438,6 +438,63 @@
...
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.p2">
+                <source>reset_password.check_email.p2</source>
+                <target><![CDATA[Если вы не получили электронное письмо, проверьте папку «Спам» или <a href="{reset_password_link}">повторите попытку</a>.]]></target>
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.help_user_exists">
+                <source>reset_password.check_email.help_user_exists</source>
+                <target>В реальном приложении письмо ниже будет отправлено, и пользователь будет перенаправлен на маршрут app_check_email.</target>
```
Comment (andrewmy) [inline translations/messages+intl-icu.ru.xlf:461]:
```suggestion
                <target>В реальном приложении это письмо было бы отправлено, и пользователь был бы перенаправлен по маршруту app_check_email.</target>
```

File: translations/messages+intl-icu.ru.xlf [hunk 3]
```diff
@@ -438,6 +438,63 @@
...
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.help_user_not_found">
+                <source>reset_password.check_email.help_user_not_found</source>
+                <target>Поскольку введенный вами адрес электронной почты не существует, мы показываем этот шаблон, вместо того, чтобы раскрывать пользователя не существует.</target>
```
Comment (andrewmy) [inline translations/messages+intl-icu.ru.xlf:465]:
```suggestion
                <target>Поскольку введенный вами адрес электронной почты не существует, мы показываем этот шаблон, вместо того, чтобы раскрывать, что пользователя не существует.</target>
```

File: translations/messages+intl-icu.ru.xlf [hunk 4]
```diff
@@ -438,6 +438,63 @@
...
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.title">
+                <source>reset_password.check_email.title</source>
+                <target>Сброс пароля Email отправлено</target>
```
Comment (andrewmy) [inline translations/messages+intl-icu.ru.xlf:469]:
```suggestion
                <target>Письмо для сброса пароля отправлено</target>
```

File: translations/messages+intl-icu.ru.xlf [hunk 5]
```diff
@@ -438,6 +438,63 @@
...
+            </trans-unit>
+
+            <trans-unit id="reset_password.email.title">
+                <source>reset_password.email.title</source>
+                <target>Привет!</target>
+            </trans-unit>
+            <trans-unit id="reset_password.email.body">
+                <source>reset_password.email.body</source>
+                <target><![CDATA[Чтобы сбросить пароль, посетите эту <a href="{email_link}">ссылку</a> истекает через {expires} час (s)..]]></target>
```
Comment (andrewmy) [inline translations/messages+intl-icu.ru.xlf:478]:
```suggestion
                <target><![CDATA[Чтобы сбросить пароль, посетите эту <a href="{email_link}">ссылку</a>. Ссылка годна ещё {expires, plural, =0 {часов} one {час} few {часа} many {часов} other {часов}}.]]></target>
```

File: translations/messages+intl-icu.ru.xlf [hunk 6]
```diff
@@ -438,6 +438,63 @@
...
+            </trans-unit>
+            <trans-unit id="reset_password.email.closing">
+                <source>reset_password.email.closing</source>
+                <target>Ура!</target>
```
Comment (andrewmy) [inline translations/messages+intl-icu.ru.xlf:482]:
```suggestion
                <target>Пока!</target>
```

File: translations/messages+intl-icu.ru.xlf [hunk 7]
```diff
@@ -438,6 +438,63 @@
...
+            </trans-unit>
+
+            <trans-unit id="reset_password.request.form_help">
+                <source>reset_password.request.form_help</source>
+                <target>Введите свой адрес электронной почты, и мы вышлем вам ссылку для сброса пароля.</target>
+            </trans-unit>
+            <trans-unit id="reset_password.request.request_btn">
+                <source>reset_password.request.request_btn</source>
+                <target>Отправка электронной почты для сброса пароля</target>
```
Comment (andrewmy) [inline translations/messages+intl-icu.ru.xlf:491]:
```suggestion
                <target>Отправить письмо для сброса пароля</target>
```

**Comment by javiereguiluz** [comment]
Closing because the related #1100 proposal was closed. Sorry! And thank you!

**Pull request closed without merging**